spec/models/cohort_spec.rb

```diff
@@ -3,6 +3,16 @@
 require "rails_helper"
 
 RSpec.describe Cohort, type: :model do
+  describe "#schedules" do
+    subject { described_class.create!(start_year: 3000) }
+
+    let!(:schedule) { create(:ecf_schedule, cohort: subject) }
+
+    it "returns associated scheules" do
+      expect(subject.schedules).to include(schedule)
+    end
```

Comment:
This test seems a bit redundant, that basically boils down to test active record. We can leave it, I'm not a massive fan of this. An alternative would be to use the shoulda matchers 

```ruby
is_expected.to bleong_to(:schedule)
```
The intent to test that the relationship is setup reads a bit clearer. But Im not fussed as it's essentially the same.

Reply:
sure. i only added as `Schedule` is namespaced under `Finance::Schedule` so wanted to confirm it